src/test/java/org/prebid/server/floors/BasicPriceFloorEnforcerTest.java

```diff
@@ -336,7 +315,7 @@ public void shouldRejectBidsHavingPriceBelowFloor() {
                         imp -> imp.id("impId2").bidfloor(BigDecimal.ONE))));
 
         final AuctionParticipation auctionParticipation = givenAuctionParticipation(
-                identity(),
+                request -> bidRequest.toBuilder(),
```

Comment:
Same here, what is going on? XD

Reply:
we need a Bid Request and a Bidder Request for the set up